adcm_client/objects.py

```diff
@@ -825,6 +825,11 @@ def run(self, **args) -> "Task":
             if 'verbose' not in args and rpm.compare_versions(
                     self.adcm_version, '2021.02.04.13') >= 0:
                 args['verbose'] = False
+            if 'verbose' in args and rpm.compare_versions(
+                    self.adcm_version, "2021.02.04.13") == -1:
+                warnings.warn(f"ADCM {self.adcm_version} doesn't support action "
+                              f"argument 'verbose'. It will be skipped")
+                args.pop('verbose')
```

Comment:
```suggestion
            if rpm.compare_versions(self.adcm_version, '2021.02.04.13') >= 0:
                args.setdefault('verbose', False)
            else:
                warnings.warn(f"ADCM {self.adcm_version} doesn't support action "
                                        f"argument 'verbose'. It will be skipped")
                args.pop('verbose', None)
```
Possible solution to the problem .